src/test/java/com/smartcar/sdk/VehicleTest.java

```diff
@@ -259,4 +282,102 @@ public void testLock() throws Exception {
 
     this.subject.lock();
   }
+
+  @Test
+  public void testBatch() throws Exception {
+    JsonObject json = Json.createObjectBuilder()
+      .add("headers", Json.createObjectBuilder()
+          .add("sc-unit-system", "metric"))
+      .add("requests", Json.createArrayBuilder()
+          .add(Json.createObjectBuilder()
+            .add("path", "/odometer")
+            .build()
+          )
+          .add(Json.createObjectBuilder()
+            .add("path", "/fuel")
+            .build()
+          )
+          .build()
+      )
+      .build();
+    RequestBody body = RequestBody.create(Vehicle.JSON, json.toString());
+    String expectedJson = "[{\"headers\":{\"unitSystem\":\"metric\"},\"path\":\"/odometer\",\"code\":200,\"body\":{\"distance\":32768}}]";
+    JsonParser parser = new JsonParser();
+    JsonElement parsed = parser.parse(expectedJson);
+    BatchResponse expectedBatch = new BatchResponse(parsed.getAsJsonArray());
+    SmartcarResponse<BatchResponse> res = new SmartcarResponse(expectedBatch);
+    ArrayList<String> paths = new ArrayList<>();
+    paths.add("/odometer");
+    PowerMockito.doReturn(res).when(this.subject, "call", eq("batch"), eq("POST"), refEq(body), refEq(BatchResponse.class));
+
+    BatchResponse batch = this.subject.batch(paths);
+
+    SmartcarResponse<VehicleOdometer> odo = batch.get("/odometer");
+    VehicleOdometer expectedOdo = new VehicleOdometer(32768);
```

Comment:
is there an easy way to add this number (32768), to the expected response that way we don't repeat the value twice?

Reply:
I'm not sure what you mean. My understanding was that this _is_ the expected response.

Reply:
This is now sort of a moot point now that the string is in separate file